kolena/asset.py

```diff
@@ -157,4 +164,29 @@ def _data_type() -> _AssetType:
         return _AssetType.AUDIO
 
 
-_ASSET_TYPES = [ImageAsset, PlainTextAsset, BinaryAsset, PointCloudAsset, BaseVideoAsset, VideoAsset, AudioAsset]
+@dataclass(frozen=True, config=ValidatorConfig)
+class Mesh3dAsset(Asset):
```

Comment:
Any reason not to call this `MeshAsset`?

Reply:
This is because the current implementation is specific to 3d mesh types, and there might be a possibility that we'd want to introduce support for 2d meshes, in which case we would want to stay consistent with [annotation naming](https://github.com/kolenaIO/kolena/blob/87a2f2d2843083c79db240b1e7cb9efc0df099f4/kolena/annotation.py#L48), for example `POLYLINE` vs `POLYLINE_3D`. I've raised https://github.com/kolenaIO/marina/pull/3960 to update UI logic.

Reply:
What is a 2D mesh?

Reply:
There could be different applications such as
- 2D finite element analysis: https://www.comsol.com/multiphysics/mesh-refinement
- Creative assets such as in Unity 2D: https://docs.unity3d.com/ScriptReference/Collider2D.CreateMesh.html

I'm coming from a more defensive angle in case we do find an use case for 2d mesh in the future, in which case the naming would otherwise be confusing if we have `MESH` for 3d and `MESH_2D` for 2d.